client/src/Movies/SavedList.js

```diff
@@ -1,4 +1,5 @@
 import React, { Component } from 'react';
+import {NavLink, Link} from 'react-router-dom';
 
 export default class SavedList extends Component {
   constructor(props) {
```

Comment:
You actually don't need this constructor at all and it is useless so remove that